lib/index.ts

```diff
@@ -8,4 +8,29 @@
 import * as Temporal from './temporal';
 import * as Intl from './intl';
 import { toTemporalInstant } from './legacydate';
+
+// Work around https://github.com/babel/babel/issues/2025.
```

Comment:
This has already been fixed in Babel - maybe we should just update our build once they release instead?

Reply:
Do we know their release schedule? 7.16.4 was almost a month ago according to GitHub, so if it's monthly and we only have to wait a few days then that seems like a good idea.

Reply:
I actually think it's somewhat unrelated to Babel's schedule, because (unless I'm missing something) anyone who uses babel while bundling their app will be vulnerable. So even if *our* bundling is OK with the new Babel, we'll still want to protect users who are using an older Babel version from being able to build code that replaces the prototype (which will break when they upgrade to the latest Babel).

BTW, we still need to merge the changes to Intl.DateTimeFormat prototype regardless of Babel.

Reply:
If we required at least 7.16.5 wouldn't users' babel update when they updated @js-temporal/polyfill to 0.3.0, as long as the versions were compatible?

Reply:
https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

Maybe I'm mis-reading this, but it seems that devDependencies (which is what our dep on babel is) aren't installed transitively, so we wouldn't be forcing downstream users to update their babel versions.

That being said, I wonder if there's a way to "suggest" that update to users to encourage they update.... maybe an (ab)use of [peerDependencies](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#peerdependencies) can do this?

Reply:
Yep, my understanding of build-time dependencies is that they're not enforced on the main app. 

Regardless, I'm not sure it's worth the bother. We understand the problem and have a simple workaround that seems to work (and is a no-op at runtime if it's been transpiled with a fixed Babel). So I'm inclined to just declare success and move on.  What do you guys think?

Reply:
That sounds reasonable - let's just merge this and move on :)